src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+const propTypes = {
+    tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    className: PropTypes.string,
+    secondary: PropTypes.bool,
+    left: PropTypes.bool,
+    right: PropTypes.bool,
+    dark: PropTypes.bool
+};
+
+const defaultProps = {
+    tag: "div"
+};
+
+const Sidebar = props => {
```

Comment:
There is an A11Y issue for this component as the `secondary` list `<div>` should be at the same level of the `primary`. Now it is:
```
<div class="sidebar-wrapper">
  <div class="sidebar-linklist-wrapper">
    <div class="link-list-wrapper">
      <ul class="link-list">
        <li></li>
        <li></li>
        <div class="linklist-secondary sidebar-linklist-wrapper"></div>
        <li></li>
      </ul>
    </div>
  </div>
</div>
```
should be like this instead:
```
<div class="sidebar-wrapper">
  <div class="sidebar-linklist-wrapper">
    <div class="link-list-wrapper">
      <ul class="link-list">
        <li></li>
        <li></li>
      </ul>
    </div>
  </div>
  <div class="linklist-secondary sidebar-linklist-wrapper">
    <div class="link-list-wrapper">
      <ul class="link-list">
        <li></li>
        <li></li>
        <li></li>
      </ul>
    </div>
  </div>
</div>
```

Reply:
Corrected it